File: cli/api/get-access-token.py
# import requests
# import json

# email='kris.nackaerts@edinsights.no'
# password='lynxvision'

# cube_id='cube-42377cfc-8e11-43ca-9e11-0f5d8f9b77d9'
# mask_id='mask-xgpjwfrj'
# project_id='project-90cc4601-a809-446f-98a5-fe33d709a849'


# auth_payload = {'username' : email, 'password' : password}
# url = 'https://maps.edinsights.no/v2/users/token'

# r=requests.post(url, data=auth_payload)
# print(r)
# print(r.json());


import requests
import json

email='kris.nackaerts@edinsights.no'
password='lynxvision'

cube_id='cube-42377cfc-8e11-43ca-9e11-0f5d8f9b77d9'
mask_id='mask-xgpjwfrj'
project_id='project-90cc4601-a809-446f-98a5-fe33d709a849'

url = 'https://maps.edinsights.no/v2/users/token?email=%s&password=%s&access_token=public' % (email,password)

r=requests.get(url)
access_token=r.json()['access_token']

# prints access token
print(access_token)

File: cli/api/get-access-token.js
var supertest = require('supertest');
var endpoints = require('./endpoints');

// domain resolution compatible with localhost setup (must run from within Docker container)
process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0" 
var MAPIC_API_DOMAIN = process.env.MAPIC_API_DOMAIN || process.env.MAPIC_DOMAIN;
var domain = (MAPIC_API_DOMAIN == 'localhost') ? 'https://172.17.0.1' : 'https://' + MAPIC_API_DOMAIN;

var domain = 'https://maps.edinsights.no';
var api = supertest(domain);
// var debug = (process.env.MAPIC_DEBUG);

var username = 'kris.nackaerts@edinsights.no';
var password = 'lynxvision';

api.post('/v2/users/token')
.send({
    username : username,
    password : password
})
.end(function (err, res) {
    if (err) console.log(err);

    // parse
    var tokens = JSON.parse(res.text);
    var access_token = tokens.access_token;

    console.log('tokens:', tokens);
    console.log('access_token:', access_token);
});